lib/appeals/configuration.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'common/client/configuration/rest'
+require 'common/client/middleware/request/camelcase'
+require 'common/client/middleware/response/caseflow_errors'
+require 'common/client/middleware/response/json_parser'
+require 'common/client/middleware/response/raise_error'
+require 'common/client/middleware/response/snakecase'
+
+module Appeals
+  class Configuration < Common::Client::Configuration::REST
+    def app_token
+      Settings.appeals.app_token
+    end
+
+    def base_path
+      "#{Settings.appeals.host}/api/v2/appeals"
+    end
+
+    def service_name
+      'AppealsStatus'
+    end
+
+    def connection
+      Faraday.new(
+        base_path, headers: base_request_headers, request: request_options, ssl: { verify: false }
```

Comment:
😕  that's copied from the original appeals service, which means it's not using it either, need to comb through the git history for why that is.

Reply:
Probably because we are hitting appeals at their AWS ELB directly, so the hostname is something like "blah-12345.elb.amazonaws.com" and they did not have a cert with a matching hostname. They don't have a public-facing URL/hostname like vets.gov does, only a VA-network-accessible endpoint, and this ELB.

But, due to the introduction of the forward proxy which terminates SSL between vets-api and backends, you can remove this. There is a cert between vets-api and the forward proxy but it should pass SSL verification.